pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -701,6 +722,12 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 				volumeToMount.VolumeSpec.Name(),
 				volumeToMount.PodName,
 				volumeToMount.Pod.UID)
+			oe.recorder.Eventf(volumeToMount.Pod, api.EventTypeNormal, kevents.SuccessfulMountVolume,
```

Comment:
`SuccessfulMountVolume` and `Unmount` will happen A LOT. Every pod (even those without volumes have a secret volume for example, and lots of volumes, like secrets, configmaps, and downwardapi remount constantly.

Therefore, I'd suggest removing these two events.